```
/var/jenkins_home/workspace/forge_forge_PR-7241/src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/InvalidModFileException.java:30: error: cannot find symbol
        super(String.format("%s (%s)", message, (ModFileInfo)modFileInfo.getFile().getFileName()));
                                                                        ^
  symbol:   method getFile()
  location: variable modFileInfo of type IModFileInfo```

--- a/src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/InvalidModFileException.java
+++ b/src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/InvalidModFileException.java
@@ -27,7 +27,7 @@ public class InvalidModFileException extends RuntimeException
 
     public InvalidModFileException(String message, IModFileInfo modFileInfo)
     {
-        super(message);
+        super(String.format("%s (%s)", message, ((ModFileInfo)modFileInfo).getFile().getFileName()));
         this.modFileInfo = modFileInfo;
     }
 }

